test/mapreduce.jl

```diff
@@ -90,10 +90,17 @@ using Statistics: mean
 
         # When the mapping and/or reducing functions are unsupported,
         # the error is thrown by `Base.mapreduce_empty`:
-        @test_throws(
-            ArgumentError("reducing over an empty collection is not allowed"),
-            mapreduce(nothing, nothing, SVector{0,Int}())
-        )
+        if Base.VERSION >= v"1.8.0-DEV.363"
+            @test_throws(
+                "reducing over an empty collection is not allowed",
+                mapreduce(nothing, nothing, SVector{0,Int}())
+            )
```

Comment:
For anyone (like myself) thinking this use of `@test_throws` looks confusing... see https://github.com/JuliaLang/julia/pull/41888